docs/man/coreos-installer-install.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: default
+parent: Command line reference
+nav_order: 1
+---
+
+# coreos-installer install
+
+## Description
+
+Install Fedora CoreOS or RHEL CoreOS
+
+## Usage
+
+* coreos-installer install [OPTIONS] &lt;device&gt;
+
+## Options
+
+* **-s**, **--stream** &lt;name&gt;  Fedora CoreOS stream
```

Comment:
These lines are pretty unreadable when rendered.  Can we use definition lists or tables?

Reply:
Converted everything to a format closer to what's currently used by podman as they use it to generate man pages (something that we may want to do even though it's not a priority).